### PR TITLE
added details about reset to 5TB

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

File: SharePoint/SharePointOnline/change-user-storage.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ As a global or SharePoint admin in Microsoft 365, you can set the OneDrive stora
 
 When you need cloud storage for individual users beyond the initial 5 TB, additional cloud storage will be granted as follows:
 
-- When a user has filled their 5 TB of OneDrive storage to at least 90% capacity, Microsoft will increase your default storage space in OneDrive to up to 25 TB per user (admins may set a lower per-user limit if they want to).
+- When a user has filled their 5 TB of OneDrive storage to at least 90% capacity, Microsoft will increase your default storage space in OneDrive to up to 25 TB per user (admins may set a lower per-user limit if they want to). The storage limit will be reset to 5TB if OneDrive storage utilization drop below 90% capacity of 5TB.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-user-storage.md:65]:
```suggestion
- When a user has filled their 5 TB of OneDrive storage to at least 90% capacity, Microsoft will increase your default storage space in OneDrive to up to 25 TB per user (admins may set a lower per-user limit if they want to). The storage limit will be reset to 5 TB if OneDrive storage utilization drops below 90% of the 5 TB capacity.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PramodBalusu, Kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PramodBalusu** [comment]
@JohanFreelancer9  any reason why this is still in open state and changes were not merged to public

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 any reason why this is still in open state and changes were not merged to public

@scanum

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**